README.md

```diff
@@ -35,7 +37,9 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+let andy = Instructor.findOne({name: "Andy"})
```

Comment:
Right idea calling `.save` on `andy`, but because of the async nature of mongo/mongoose, the `andy` variable doesn't contain the document outside of the `findOne` callback:
```js
// Assuming we have model for wishlist_items, named WishItem
Instructor.findOne({name: "Andy"}, (err, andy)=>{
    andy.wishlist_items.push(new WishItem({description: "Resin Laying Deer Figurine, Gold"}))
    andy.save();
}
```